dotnet/samples/Demos/ProcessFrameworkWithAspire/ProcessFramework.Aspire/ProcessFramework.Aspire.AppHost/Program.cs

```diff
@@ -0,0 +1,17 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+var builder = DistributedApplication.CreateBuilder(args);
+
+var openai = builder.AddConnectionString("openAiConnectionName");
+
+var translateAgent = builder.AddProject<Projects.ProcessFramework_Aspire_TranslatorAgent>("translatoragent")
```

Comment:
the service names "translatoragent" and "summaryagent" are referenced in multiple places. Maybe move them to constants under ServiceDefaults?

Reply:
`translatoragent` and `summaryagent` are the names of the aspire resources. They should only be referenced here. I don't recall having referenced those names elsewhere...

Reply:
they are referenced here dotnet/samples/Demos/ProcessFrameworkWithAspire/ProcessFramework.Aspire/ProcessFramework.Aspire.ProcessOrchestrator/Program.cs for example